app/src/main/java/org/gdg/frisbee/android/activity/AppInviteDeepLinkActivity.java

```diff
@@ -44,9 +45,7 @@
 
     @Override
     protected GoogleApiClient createGoogleApiClient() {
-        return new GoogleApiClient.Builder(this)
-            .addApi(AppInvite.API)
-            .build();
+        return GoogleApiClientFactory.createWithoutSignIn(getApplicationContext());
```

Comment:
I am not sure if it is a good practice to create GoogleApiClient with ApplicationContext. 

GoogleApiClient is really tied to the Activity. We are recreating it on every Activity. That's how it needs to be done. So I am not sure if this makes sense.